sci-physics/jaxodraw-bin/jaxodraw-bin-2.1.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2 desktop latex-package
```

Comment:
Usually we sort these alphabetically, unless there is a specific reason to use a different inherit order (e.g. exported functions that would otherwise be overridden)